### ISSUE TITLE
List export as PDF, file corrupt if list name with special characters

### ISSUE THREAD

**Issue by eingutertag** [Admidio/admidio#702]
If you export a list to PDF with special characters in the list name (i.e. Members 2017/18) the resulting PDF file is corrupt.
As a workaround I replaced the character "/" in the **show_list.php** function with "_".
Perhaps you could find a better solution for the issue?

str_replace('.', '', **str_replace('/', '_', $roleName)**)
```
// Settings for export file
// Replace Slash (/) in PDF filenames
if ($getMode === 'csv' || $getMode === 'pdf')
{
    // file name in the current directory...
    if (strlen($list->getValue('lst_name')) > 0)
    {
        $filename = $gCurrentOrganization->getValue('org_shortname').'-'.str_replace('.', '', str_replace('/', '_', $roleName)).'-'.str_replace('.', '', $list->getValue('lst_name')).'.'.$getMode;
    }
    else
    {
        $filename = $gCurrentOrganization->getValue('org_shortname').'-'.str_replace('.', '', str_replace('/', '_', $roleName)).'.'.$getMode;
    }

```

**Comment by ximex** [comment]
There is a pull request coming that has a function for that:
https://github.com/Admidio/admidio/blob/feature-use-filesystemutils/adm_program/system/classes/filesystemutils.php#L56
We should check all code where filenames are build and safe them with this function

**Comment by ximex** [comment]
Should be fixed in bfe3527e62cae249df7d2e36a94d07928e884abe

**Comment by ximex** [comment]
@eingutertag please test when the v3.3 beta is released an reopen the issue if the bug still exist
